### DISCUSSION TITLE
Google Chromes phase out of 3rd Part Cookies

### DISCUSSION THREAD

**Discussion by trv-bmpage** [argoproj/argo-cd#16362] [Q&A] [answered]
https://developer.chrome.com/en/docs/privacy-sandbox/third-party-cookie-phase-out/ 

Is there any concern around accessing the Argo CD UI via chrome after 3rd party cookie phase out?

**Comment by FernandoMiguel** [comment, marked as answer]
i've had 3rd party cookies turned off on chrome for many years.
i can work in argo just fine, login via SSO , no issues
